pom.xml

```diff
@@ -1419,16 +1412,6 @@
     </profile>
 
     <!-- profile that Alluxio developers should use -->
-    <profile>
-      <id>developer</id>
-      <modules>
-        <module>integration/yarn</module>
-      </modules>
-      <properties>
-        <hadoop.version>3.3.0</hadoop.version>
-      </properties>
-    </profile>
-
```

Comment:
Should I not remove all of these but only 1424-1426?

Reply:
@Xenorith  do you know which script is calling "developer" profile?

Reply:
I found `dev/jenkins/build.sh`, `dev/github/run_tests.sh`, `dev/github/run_checks.sh` include commands to run `mvn` with `-Pdeveloper`.

Reply:
Please keep the developer profile, and remove the yarn part.

Reply:
it may be possible to remove it, but it should be done in a separate PR. it looks like it was previously used to indicate specific modules and set `hadoop.version`, but this PR would remove the last module. the `hadoop.version` is matching the one defined in the root pom.xml's list of properties, so i believe the profile ends up doing nothing.

Reply:
Added back developer profile but removed the module tag.